bin/xspec.bat

```diff
@@ -120,6 +121,9 @@ rem ##
         set XSLT=1
     ) else if "%WIN_ARGV%"=="-q" (
         set XQUERY=1
+    ) else if "%WIN_ARGV%"=="-s" (
+        set SCHEMATRON=1
+        set XSLT=1

```

Comment:
* `SCHEMATRON` should be reset at `:win_reset_options`.
  * The purpose of `:win_reset_options` is to negate the environmental variables when they are already set outside the batch.

* `XSLT` should not be set here.
  * Setting `XSLT` here makes it impossible to tell "-s and -t are mutually exclusive".
  * `XSLT` is set [later](https://github.com/xspec/xspec/blob/f4811a5976c637210e651f5b46d962f38d183f9b/bin/xspec.bat#L310) anyway.